riscv/tests/riscv.rs

```diff
@@ -13,13 +12,12 @@ use test_log::test;
 
 use powdr_riscv::{
     continuations::{rust_continuations, rust_continuations_dry_run},
-    Runtime,
+    CompilerOptions, RuntimeLibs,
 };
 
 /// Compiles and runs a rust program with continuations, runs the full
 /// witness generation & verifies it using Pilcom.
 pub fn test_continuations(case: &str) {
```

Comment:
should these tests be `_64`? or should we also have BB inside here?

Reply:
or maybe  `_gl`?

Reply:
eventually we should have bb as well, but for now only 64. Do you mean the test name?

Reply:
yes, was just thinking if we should have different `test_continuations_foo` tests or just test all variants in a single test

Reply:
will probably add a different one at that point, but I'd wait until we get there to change it